### PR TITLE
Feature/handle cmp list

### PR DESCRIPTION
**Pull request by srinivas81**

### CODE REVIEW

File: iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Cmp.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+import java.time.Instant;
+import java.util.Optional;
+
+public class Cmp implements com.iabtcf.extras.Cmp {
+
+    private int id;
+    private String name;
+
+    @JsonProperty("isCommercial")
```
Comment (laktech) [inline iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Cmp.java:33]: why does this attribute need the JsonProperty tag?

File: iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Cmp.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
+package com.iabtcf.extras.jackson;
```
Comment (laktech) [inline iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Cmp.java:1]: this package name should reflect whether this is gvl or cmp

File: iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Feature.java [hunk 3]
```diff
@@ -1,8 +1,8 @@
-package com.iabtcf.gvl.jackson;
+package com.iabtcf.extras.jackson;
```
Comment (laktech) [inline iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Feature.java:1]: package name should reflect gvl

File: iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Loader.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        return objectMapper.readValue(json, com.iabtcf.extras.Gvl.class);
+    }
+
+    /**
+     * Gets the CMP list from the specified InputStream
+     *
+     * @param content {@link InputStream} of CMP List json content
+     * @return {@link com.iabtcf.extras.CmpList} object
+     */
+    public com.iabtcf.extras.CmpList loadCmpList(InputStream content) throws IOException {
```
Comment (laktech) [inline iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Loader.java:83]: add loadCmpList(String) for consistency

File: iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Loader.java [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+        objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+    }
+
+    /**
+     * Gets the global vendor list json from the specified InputStream
+     *
+     * @param content {@link InputStream} of GVL json content
+     * @return {@link com.iabtcf.extras.Gvl} object
+     */
+    public com.iabtcf.extras.Gvl loadGlobalVendorList(InputStream content) throws IOException {
```
Comment (laktech) [inline iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Loader.java:63]: "load" prefix is redundant, it's already in the name of the class. "globalVendorList" is sufficient.

File: iabtcf-extras/src/main/java/com/iabtcf/extras/CmpList.java [hunk 6]
```diff
@@ -0,0 +1,50 @@
+package com.iabtcf.extras;
```
Comment (laktech) [inline iabtcf-extras/src/main/java/com/iabtcf/extras/CmpList.java:1]: add cmp to package name

File: iabtcf-extras/src/main/java/com/iabtcf/extras/Feature.java [hunk 7]
```diff
@@ -1,8 +1,8 @@
-package com.iabtcf.gvl;
+package com.iabtcf.extras;
```
Comment (laktech) [inline iabtcf-extras/src/main/java/com/iabtcf/extras/Feature.java:1]: add gvl to package name

File: iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Cmp.java (hunk 1 above)
Comment (srinivas81) [reply iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Cmp.java:33]: The field name in the JSON schema itself is IsCommercial. Unless I change my field name to isIsCommercial (two  "is"), the  jackson parser will not work. Hence I need to tell the jackson parser that the property name is isCommercial.

Comment (laktech) [reply iabtcf-extras-jackson/src/main/java/com/iabtcf/extras/jackson/Cmp.java:33]: that makes sense. thanks for explaining

**Pull request merged**